src/components/admin/create/events/AddEventPage.js

```diff
@@ -111,15 +134,15 @@ export const AddEventPage = () => {
         <div className="hidden md:flex md:w-1/6 lg:w-1/3" />
         <div className="md:w-4/6 lg:w-1/3">
           <p className="relative text-xl font-bold text-hoboken-blue left-4">
-            Hi {user.firstName}, lets create a happening
+            Hi {user.firstName}, let's create an event
           </p>
           <ErrorText extraProps={"ml-4"}>
-            {Object.keys(error).length > 0 ? {error} : null}
+            {Object.keys(error).map(e => <div>{error[e]}</div>)}
```

Comment:
This is great, but if possible, putting the corresponding errortext to each specific field would help to give context for a user. I.e Putting "Location Name is required" above or under the input itself. Otherwise this looks great!